lib/btrfs_test.pm

```diff
@@ -72,6 +74,17 @@ sub snapper_nodbus_restore {
     die 'DBus service should be inactive, but it is active' if ($ret == 0);
     script_run('systemctl default', timeout => 600, die_on_timeout => 0);
     my $tty = get_root_console_tty;
+
+    if (is_sle('<15-SP3') && !defined(my $match = check_screen("tty$tty-selected", 120))) {
+        record_soft_failure("bsc#1185098 - [Build 15.82] openQA test fails in snapper_undochange - logind fails after return back from rescue");
```

Comment:
nitpicking, no need to provide full title of the bug, but feel free to reject :)
```suggestion
        record_soft_failure("bsc#1185098 - logind fails after return back from rescue");
```

Reply:
Nah, I that was just my lazy copy&paste. Updated :)